### PR TITLE
RM-7510: Firefox: Left edge of the leftmost button is cut off

### PR DESCRIPTION
**Pull request by iam-mholle**
https://re-motion.atlassian.net/browse/RM-7510

### CODE REVIEW

**Comment by iam-mholle** [comment]
## Layout changes
### Before
FF_before_fix

### After
FF_after_fix

### Diff
FF_diff

**Comment by iam-mholle** [comment]
The space between the button and the following text needs to be increased.

**Comment by iam-mholle** [comment]
### A default HTML Button for reference

image

File: Remotion/Web/Core/Res/Themes/NovaGray/HTML/WebButton.css [hunk 1]
```diff
@@ -96,3 +96,8 @@ button.webButton[disabled] span.buttonBody img
   filter: progid:DXImageTransform.Microsoft.BasicImage(opacity=0.40),
           progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
 }
+
+.mozilla button.webButton span.buttonBody
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Res/Themes/NovaGray/HTML/WebButton.css:100]: wouldn't the -2px be a bug that should be fixed globally isntead of just for Firefox?

Comment (iam-mholle) [reply Remotion/Web/Core/Res/Themes/NovaGray/HTML/WebButton.css:100]: In Chrome (for example), the `left: -2px;` setting is needed to make sure WebButtons and normal buttons are aligned.

### Chrome with `left: -2px`
image

In Firefox however, the setting breaks the alignment now.

### Firefox with `left: -2px`
image

Comment (iam-mholle) [reply Remotion/Web/Core/Res/Themes/NovaGray/HTML/WebButton.css:100]: Interesting, as you can see in the above pictures, buttons in chrome do not span over the whitespace after them, in Firefox they do.

Comment (MichaelKetting) [reply Remotion/Web/Core/Res/Themes/NovaGray/HTML/WebButton.css:100]: headache. next week pairsession?

**Comment by iam-mholle** [comment]
# Layout with adapted Button width/padding
## Firefox

image

## Chrome

image

File: Remotion/Web/Core/Res/Themes/NovaGray/HTML/WebButton.css [hunk 2]
```diff
@@ -96,3 +96,11 @@ button.webButton[disabled] span.buttonBody img
...
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Res/Themes/NovaGray/HTML/WebButton.css:100]: Mozilla...oooookay....

**Review by MichaelKetting — APPROVED** [review]

**Comment by iam-mholle** [comment]
Added to master with 100ef42c3827a09f45e89d70a1569e800aece307.

**Pull request closed without merging**